Gordon360/ApiControllers/AccountsController.cs

```diff
@@ -306,7 +332,7 @@ public IHttpActionResult SearchWithSpace(string searchString, string secondarySt
                 }
                 precedence++;
 
-                // Nickname and last name start with (Lowest Priority)
+                // Nickname and last name start with

```

Comment:
This comment needs to be finished, I think.

Reply:
Check line 313